lib/gc/Transforms/Pipeline.cpp

```diff
@@ -53,6 +53,8 @@ void populateTensorPasses(mlir::OpPassManager &pm) {
   // todo: padding propagation pass
   // todo: layout propagation pass
   // todo: tensor constant propagation pass
+  pm.addPass(createConstantSubgraphAnalysisPass());
+  pm.addPass(createConstantTensorFoldingPass());
```

Comment:
Shall we combine these two passes into one, and provide an option to do the analysis only is needed?

Reply:
I feel the same. Maybe we should put the analysis into the pass, unless the const-subgraph-analysis is needed by more than one pass.

Reply:
OK, I will put them into to one.